homeassistant/components/lutron/manifest.json

```diff
@@ -5,5 +5,9 @@
   "requirements": ["pylutron==0.2.8"],
   "codeowners": ["@JonGilmore"],
   "iot_class": "local_polling",
-  "loggers": ["pylutron"]
+  "loggers": ["pylutron"],
+  "supported_brands": {
+    "lutron_homeworks_qs": "Lutron HomeWorks QS",
+    "lutron_ra2": "Lutron RA2"
```

Comment:
This integration is called "Lutron", how is adding product(lines) as supported brand going  to help?

It is the same brand already? Isn't this misusing the supported brands feature?

Reply:
Lutron has some confusing branding and its difficult for users to figure out which integration to use:

Lutron Caséta Wireless which LEAP protocol (aka `lutron_caseta` integration)
Lutron Caséta Pro which LEAP protocol (aka `lutron_caseta` integration)
Lutron RA2 which uses the LIP protocol (aka `lutron` integration)
Lutron RA2 Select which uses the LEAP protocol (aka `lutron_caseta` integration)
Lutron RA3 which uses the LEAP protocol (aka `lutron_caseta` integration)
Lutron HomeWorks QS which uses the LIP protocol (aka `lutron` integration)
Lutron HomeWorks QSX which uses the LEAP protocol (aka `lutron_caseta` integration)